src/util/configHelpers.js

```diff
@@ -675,6 +676,35 @@ const restructureListingFields = hostedListingFields => {
 // Merge listing configs //
 ///////////////////////////
 
+const validateCategoryConfig = (hostedConfig) => {
```

Comment:
Add a code comment that explains what this looks like (since there isn't a built-in file for the data).
Of course, validation code gives away this structure too, but people grasp the code better when the format is explicitly shown.

```js
// The expected structure of the hosted config:
// {
//   categories: [
//     {
//       name: 'Cats',
//       id: 'cats',
//       subcategories: [{ name: 'Burmese', id: 'burmese' }, { name: 'Egyptian Mau', id: 'egyptian-mau' }],
//     },
//     // and so on.
//   ],
// }
```